setup.cfg

```diff
@@ -38,9 +38,9 @@ setup_requires =
     wheel
 install_requires =
     colorama>=0.3.4
-    backports.functools_lru_cache;python_version<="3.4"
-    backports.shutil_get_terminal_size;python_version<"3.3"
-    backports.weakref;python_version<"3.3"
+    backports.functools_lru_cache;python_version=="2.7"
+    backports.shutil_get_terminal_size;python_version=="2.7"
+    backports.weakref;python_version=="2.7"
```

Comment:
Are these truly only supported on python 2.7 or is this done to indicate that only python 2.7 is supported? More of a nitpick and it won't hold the PR back IMO

Reply:
Those backports very likely work correctly with the constraints that you were using, but it means you need to correlate these versions with the ones in compat.py - if the versions dont match up, a library could be installed and then not used, or installed and used when the stdlib version is better, which is what was happening with https://github.com/sarugaku/vistir/issues/63

As the vistir library has `python_requires = >=2.6,!=3.0,!=3.1,!=3.2,!=3.3` in setup.cfg , there is no benefit in caring about whether these work correctly on py32,py33,py34 etc.  vistir only needs them for py27.  'KISS'.